### DISCUSSION TITLE
Adding an option to pin lists to the timeline

### DISCUSSION THREAD

**Discussion by binkelderg** [insin/control-panel-for-twitter#84] [Ideas]
Ever since Twitter Lists Redux stopped working because Twitter updated its UI, not a single alternative has come out. The lists feature is terrible as is; you can't really organize or sort them in any way, nor can you even pin them to the timeline on desktop like you can on mobile. It's just cluttered garbage at this point. 

I think me and many others would greatly appreciate it if anyone could try to improve that entire experience, because Twitter doesn't give a shit.

**Comment by ka3l-bit** [comment]
it would be great if we could come back to this! i'd love to at least be able to be automatically redirected to lists from home instead of messages or notifications, and removing retweets from list timelines would also be great

**Comment by Rust-NeoShiny** [comment]
I'd also love this. I have quite a few lists and I want to be able to go into them without having to go through menus,

**Comment by marypcbuk** [comment]
I would *love* to be able to pin a list at the top level of the navigation!

**Comment by insin** [comment]
Twitter enabled this feature on the web version when they moved to the tabbed timeline:

![firefox_BmPF52jFD5](https://user-images.githubusercontent.com/226692/216558935-9853cd5f-507f-47fd-b88e-73e2185d84d8.png)

![firefox_6yvlfJFxAg](https://user-images.githubusercontent.com/226692/216559026-8e8d7ac1-2d8f-4426-b8b0-329df2afd792.png)
